frontend/src/pages/Graph/__tests__/SummaryLink.test.tsx

```diff
@@ -29,6 +41,9 @@ describe('renderBadgedLink', () => {
         <MemoryRouter>{renderBadgedLink(node)}</MemoryRouter>
       </Provider>
     );
+    debugger;
+    console.log('change');
+    console.log(wrapper.find('a').debug());
```

Comment:
Did you want these?

Reply:
No clearly I was fighting with my debugger